src/software/amazon/ion/impl/LocalSymbolTableImports.java

```diff
@@ -74,49 +74,14 @@
      */
     LocalSymbolTableImports(List<SymbolTable> importTables)
     {
-        validateImports(importTables);
-
         int importTablesSize = importTables.size();
 
-        // detects and adapts local tables so they are importable
-        myImports = new SymbolTable[importTablesSize];
-        for(int i = 0; i < importTables.size(); i++)
-        {
-            SymbolTable symbolTable = importTables.get(i);
-            if(symbolTable.isLocalTable())
-            {
-                myImports[i] = LocalSymbolTableImportAdapter.of((LocalSymbolTable) symbolTable);
-            }
-            else
-            {
-                myImports[i] = symbolTable;
-            }
-        }
+        myImports = importTables.toArray(new SymbolTable[importTablesSize]);
```

Comment:
The real reason is because the changes to LocalSymbolTableImports in this PR simply revert the changes applied to it in #111. As to which version of `toArray` to use, I do prefer this version both because it avoids a cast and because it shows very explicitly that an allocation is being done.